### ISSUE TITLE
Allow othername & lineage

### ISSUE THREAD

**Issue by sknorr** [openSUSE/geekodoc#83]
Within \<personname/\>s, currently only \<firstname/\> & \<surname/\> are allowed. However, a bunch of people legitimately do have middle names or Jr./Sr. added to their name. Hence, \<othername/\> and \<lineage/\> should probably be allowed. (AsciiDoctor also tends to generate those tags.)

Should I ever have argued for the opposite, please ignore my past opinions. :)

**Comment by tomschr** [comment]
Removing `<firstname>` will cause validation problems. I would like to avoid these. Therefor I try to keep it for the time being and add `<givenname>`, `<lineage>`, and `<honorific>` to GeekoDoc.

**Comment by sknorr** [comment]
If we still operate under the assumption that people have GeekoDoc v1 and v2 installed side by side, I think this might be a compatibility break worth making -- people would explicitly update to the new schema and it would be expected that there may be a few minor issues like this. No?

**Comment by tomschr** [comment]
Good point. In that sense, a major release *could* introduce a compatibility break. The question is if it's worth.

After I've looked into the doc-sle repo, it isn't not that much: I've counted 16 files which contains firstname. So it seems manageable...

Let me think about it.

**Comment by sknorr** [comment]
The major repo where the tag is relevant is going to be the SBP repo, I guess. But that largely uses plain DocBook because the documents there are incompatible for a number of reasons.

**Comment by tomschr** [comment]
Effort is probably low, could add the other tags.
